frontend/mulighetsrommet-cms/schemas/tiltakstype.js

```diff
@@ -44,15 +52,43 @@ export default {
       title: "Varighet",
       type: "string",
     },
+    //TODO skal kunne legge til flere lenker og filer
     {
       name: "regelverkFil",
       title: "Regelverk fil",
       type: "file",
     },
+    {
+      name: "regelverkFilNavn",
+      title: "Navn til fil",
+      description: "Hvilket navn skal vises til filen?",
+      type: "string",
+      validation: (Rule) =>
+        Rule.custom((field, context) =>
+          context.document.regelverkFil && field === undefined
+            ? "This field must not be empty."
+            : true
+        ),
+      hidden: ({ document }) => !document?.regelverkFil,
+    },
     {
       name: "regelverkLenke",
       title: "Regelverk lenke",
       type: "url",
+      placeholder: "https://www...",
+    },
+    {
+      name: "regelverkLenkeNavn",
+      title: "Navn til lenke",
+      description: "Hvilket navn skal vises i lenken?",
+      type: "string",
+      validation: (Rule) =>
+        Rule.custom((field, context) =>
+          context.document.regelverkLenke && field === undefined
+            ? "This field must not be empty."
```

Comment:
Samme her for engelsk. Endre til norsk? Antar dette dukker opp i gui i sanity? Ikke superkritisk siden det er for oss utviklerne, men kan være greit å være konsekvent i språket?